doc(release): prepare release notes for Centreon Web 22.10.3 (#2064)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md

## Centreon Web

### 22.10.3

Release date: `soon`

#### Enhancements

- [APIv2] Added vault configuration endpoint

#### Bug fixes

- Fixed a bug that blocked exporting multiple pollers' configuration at the same time with a shared severity
- [Authentication] Fixed authentication with complete URL for token endpoint
- Fixed bug where it's impossible to filter on hosts and/or services in Centreon 22.10.
- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
- Allow users to configure Broker's global maximum retention
- Fixed an issue that caused months to be missing from the calendar selection
- Fixed bug with pagination on event log page
- Fixed recurrent downtime editing issues with disabled objects

> - [Resource-Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resource Status page

Reviewer: ```suggestion
- [Resources Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resources Status page
```

## Centreon Web

### 22.10.3

> Release date: `January 2023, 9`

Reviewer: ```suggestion
Release date: `January 9, 2023`
```

- Fixed bug with pagination on event log page
- Fixed recurrent downtime editing issues with disabled objects
- [Resource-Status] Fixed an issue causing ACL action "Display executed command by monitoring engine" to not be applied on the Resource Status page
- Fixed "view contact notifications" window
- [API] Fixed API access when user doesn't have access to UI
- Fixed an issue where the pagination was not displayed in all legacy pages

#### Security fixes

- [Core] Fixed vulnerabilities in functions.js file
- [Configuration] Sanitized queries in the list of services by host group
- [Configuration] Sanitized queries in the list of host categories
- [Configuration] Sanitized queries in the list of commands
- [Core] Fixed vulnerabilities in ajaxLdapSearch.js file
- [Configuration] Sanitized queries in the list of Broker configurations
- [Core] Fixed vulnerabilities in color_picker.php
- [Core] Fixed vulnerabilities in pathway.php
- [Core] Fixed vulnerabilities in color_picker_mb.php
- [Core] Fixed vulnerabilities in rename.php
- [Configuration] Fixed vulnerabilities in services listing
- [Configuration] Fixed vulnerabilities in host form

> - Fixed issue pagination not displayed in all legacy pages

Reviewer 2: This line is already in the Bug fixes part (line 44)

## Centreon Web

### 22.10.3

Release date: `soon`

#### Enhancements

- [APIv2] Added vault configuration endpoint

#### Bug fixes

> - Fixed a bug that blocked exporting multiple pollers' configuration at he same time with a shared severity

Reviewer: ```suggestion
- Fixed a bug that blocked exporting multiple pollers' configuration at the same time with a shared severity
```

> - [Authentication] Fixed authentication with complete url for token endpoint

Reviewer: ```suggestion
- [Authentication] Fixed authentication with complete URL for token endpoint
```

> - Fixed bug whre it's impossible to filter on hosts and/or services in 22.10.

Reviewer: ```suggestion
- Fixed bug where it's impossible to filter on hosts and/or services in Centreon 22.10.
```

- Fixed an ACL bug allowing users to modify users even when they had read-only permissions
- Acknowledgement/downtimes failed without returning an error when the character case for the resource was wrong
- Fixed a bug linked to forbidden characters in Engine object names that could strip the '0', '3' and '9' digits from host names
- Allow users to configure Broker's global maximum retention
- Fixed an issue that caused months to be missing from the calendar selection

> - Bugfix of pagination on event log page

Reviewer: ```suggestion
- Fixed bug with pagination on event log page
```

> - Recurrent downtime editing issues with disabled objects

Reviewer: ```suggestion
- Fixed recurrent downtime editing issues with disabled objects
```

> - [Resource-Status] Monitoring engine command displayed in detail regarding ACL configured

Reviewer: what does the sentence mean?

> - Fix view contact notfications window

Reviewer: ```suggestion
- Fixed "view contact notifications" window
```

- [API] Fixed API access when user doesn't have access to UI

> - Fixed issue pagination not displayed in all legacy pages

Reviewer: ```suggestion
- Fixed an issue where the pagination was not displayed in all legacy pages
```

#### Security fixes

- [Core] Fixed vulnerabilities in functions.js file
- [Configuration] Sanitized queries in the list of services by host group
- [Configuration] Sanitized queries in the list of host categories
- [Configuration] Sanitized queries in the list of commands
- [Core] Fixed vulnerabilities in ajaxLdapSearch.js file

> - [Configuration] Sanitized queries in the list of broker configurations

Reviewer: ```suggestion
- [Configuration] Sanitized queries in the list of Broker configurations
```

- [Core] Fixed vulnerabilities in color_picker.php
- [Core] Fixed vulnerabilities in pathway.php
- [Core] Fixed vulnerabilities in color_picker_mb.php
- [Core] Fixed vulnerabilities in rename.php
- [Configuration] Fixed vulnerabilities in services listing
- [Configuration] Fixed vulnerabilities in host form
- [API] Fixed API access when user doesn't have access to UI

> - Fixed issue pagination not displayed in all legacy pages

Reviewer: already appears above

## Centreon Web

### 22.10.3

Release date: `January 9, 2023`

> Un problème connu affecte actuellement la version 22.10.3. Patientez avant de mettre votre plateforme à jour : une nouvelle version va être publiée sous peu.

Reviewer: ```suggestion
> Un problème connu affecte actuellement la version 22.10.3. Patientez avant de mettre votre plateforme à jour : une nouvelle version va être publiée sous peu.
```
